app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -327,7 +323,7 @@ protected String getTrackedViewName() {
         } catch (IndexOutOfBoundsException e) {
             pageName = "";
         }
-        Chapter chapter = mChapterAdapter.findById(selectedChapterId);
+        Chapter chapter = findChapterById(selectedChapterId);
```

Comment:
this could use chapter as a variable instead of searching for the chapter for each track event (not part of this PR though)